Replace built-in sinks with modules (#576)
eik-lib/service: README.md

> # Eik HTTP Service
> 
> Eik REST API - As a standalone running HTTP service
> 

Reviewer: ```suggestion
# Eik HTTP Service

As a standalone running HTTP service exposing the Eik Service.

```
I found it confusing to have the "rest"-bit in there

## Installation

```
npm install @eik/service
```

## Usage

This server can either be run as a Node executable, or as a Fastify plugin.

```sh
npx @eik/service
```

> For a production setup, the Fastify plugin method is recommended.

Reviewer: How about splitting this into two?
one simple section showing how to start it, without a separate http server using `bin` and then another section where your mount the `.api()` as a Fastify plugin.

```js
import fastify from 'fastify';
import Service from '@eik/service';

> import Sink from '@eik/sink-gcs';
> 
> // Set up the Google Cloud Storage sink
> // https://github.com/eik-lib/sink-gcs?tab=readme-ov-file#example
> const sink = new Sink({
>     credentials: {
>         client_email: 'a@email.address',
>         private_key: '[ ...snip... ]',
>         projectId: 'myProject',
>     },
> });

Reviewer: ```suggestion
import GcsSink from '@eik/sink-gcs';

// Set up the Google Cloud Storage sink
// https://github.com/eik-lib/sink-gcs?tab=readme-ov-file#example
const sink = new GcsSink({
    credentials: {
        client_email: 'a@email.address',
        private_key: '[ ...snip... ]',
        projectId: 'myProject',
    },
});
```
Naming the default import makes it easier to read and grok what's happening I feel.

// Set up the Eik service as a plugin
const service = new Service({ sink });

// Set up Fastify
const app = fastify({
    ignoreTrailingSlash: true,
    modifyCoreObjects: false,
    trustProxy: true,
});

// Register the Eik service in Fastify
app.register(service.api());

> // Append custom HTTP ready checks
> app.get('/_/health', (request, reply) => {
>     reply.send('ok');
> });
> 
> app.get('/_/ready', (request, reply) => {
>     reply.send('ok');
> });

Reviewer: ```suggestion
```
I don't feel this is important to have here, people can do what they want here and those who require ready checks would know what to do.
